### ISSUE TITLE
Hiding Relay top and bottom buttons

### ISSUE THREAD

**Issue by pelted** [sibartlett/homebridge-wink3#75]
Is there an exposed way to easily hide the Relay top and bottom buttons? There really is not reason I want these showing up as devices in HomeKit and have had to find creative names for them since they were colliding with actual devices when using Siri to toggle things on and off. I guess maybe device ids, but could these be lumped into a group to be used by hide_groups?

**Comment by sibartlett** [comment]
The two switches that show up in HomeKit, are not the buttons on the Relay, but rather the wired connections on the back of the unit. Do you not have any circuits wired up to the Relay?

**Comment by pelted** [comment]
Correct. They are circuits when wired as such. Some are some aren’t. I just went ahead and hid the ids for the ones showing up. The issue comes when you have them setup in “Smart” mode which is supposed to keep the load on and enable the buttons to be used to control anything. This causes the wired connections to show in HomeKit when really you wouldn’t want them too since turning them off causes the load to switch off even when in this mode. That is a problem with the Relay really. I have several wired this way for a couple of reasons but may rewire later.

**Comment by sibartlett** [comment]
Gotcha. I haven't got a Wink Relay, so I'm not sure if the Wink API has some indicator whether the relay is in Smart mode.
